### DISCUSSION TITLE
code search can only shows results before oct.15

### DISCUSSION THREAD

**Discussion by deezertidal** [community/community#36494] [Other Feature Feedback, Questions, & Ideas] [answered]
is it a bug?

**Comment by Akash1134** [comment]
Hi @deezertidal , We are happy you are here! Please share some more details around the search operation. Does the problem occurs with a specific project/repo/org? Also, are you noticing a delay in the recent PR's being un-indexed in the search?

**Reply by AngmarLord** [reply]
I have a feeling he's referring to the fact that GitHub universal code search is not showing any recent result, for any keyword. All results are October 15 or earlier, even for popular keywords.

There's another discussion topic here - 
https://github.com/orgs/community/discussions/36514

**Reply by deezertidal** [reply]
you can try all keywords in all languages，it will never show the results after oct15. the newest result is 4days ago ，and tomorrow it will be 5days ago

**Reply by pl0mo** [reply]
I am experiencing the same issue, all code searchs ordered by indexed return results indexed > 4 days ago.

**Comment by deezertidal** [comment, marked as answer]
fixed！

**Reply by look** [reply]
Yes, sorry about the issue, but it has been addressed.
